content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -86,13 +90,17 @@ For NLog version 4.5:
 {{< /tabs >}}
 
 
-## Manual Trace ID injection
+## Manual log injection
 
-If you prefer to manually correlate your [traces][6] with your logs, leverage the Datadog API to retrieve correlation identifiers:
+If you prefer to manually correlate your [traces][6] with your logs and tie together data for your service,
+leverage the Datadog API to retrieve correlation identifiers:
 
-- Use `CorrelationIdentifier.TraceId` and `CorrelationIdentifier.SpanId` API methods to inject identifiers at the beginning and end of each [span][7] to log (see examples below).
+- Use `CorrelationIdentifier.<FIELD>` API methods to inject identifiers at the beginning and end of each [span][7] to log (see examples below).
 - Configure MDC to use the injected keys:
 
+    - `dd.env` Globally configured `env` for the tracer (or `""` if not set)
+    - `dd.service` Globally configured root service name (or `unnamed-dotnet-service` if not set)
+    - `dd.version` Globally configured `version` for the service (or `""` if not set)
     - `dd.trace_id` Active Trace ID during the log statement (or `0` if no trace)
     - `dd.span_id` Active Span ID during the log statement (or `0` if no trace)
 
```

Comment:
I'm unable to comment directly on the Serilog lines below because you didn't initially touch on them, but can you place the following lines above the the line containing `using (LogContext.PushProperty("dd.trace_id", CorrelationIdentifier.TraceId.ToString()))`:

```
using (LogContext.PushProperty("dd.env", Tracer.Instance.Settings.Environment))
using (LogContext.PushProperty("dd.service", Tracer.Instance.DefaultServiceName))
using (LogContext.PushProperty("dd.version", Tracer.Instance.Settings.ServiceVersion))
```